src/Symfony/Component/Security/Csrf/composer.json

```diff
@@ -25,7 +25,7 @@
         "symfony/http-foundation": "^2.7.38|~3.3.13"
     },
     "conflict": {
-        "symfony/http-foundation": "<2.7.38|~2.8,<2.8.31|~3.3,<3.3.13|~3.4,<3.4-beta5"
+        "symfony/http-foundation": "<2.7.38|~2.8,<2.8.31|~3.3,<3.3.13"
```

Comment:
Didn't you forget to add `~3.4` instead?

Reply:
that's a conflict rule, it works inversed: we do allow 3.4 (after beta5)